client-projects/the-lumery/filterButtons.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  FILTER_BUTTON_LIST_SELECTOR,
+  FILTER_RESET_BUTTON_SELECTOR,
+} from "./selectors";
+
+export function setUpFilterButtons() {
+  const resetButton: HTMLElement | null = document.querySelector(
+    FILTER_RESET_BUTTON_SELECTOR,
+  );
+  if (!resetButton) {
+    throw new Error(
+      `Reset button element not found: ${FILTER_RESET_BUTTON_SELECTOR}`,
+    );
+  }
+  const filterButtonList: HTMLElement | null = document.querySelector(
+    FILTER_BUTTON_LIST_SELECTOR,
+  );
+  if (!filterButtonList) {
```

Comment:
I've just realised that I can do the [`instanceOf HTMLElement` check](https://github.com/springload/webflow-dev-workflow/pull/14#discussion_r1382635964) here as well to tidy things up a bit.